src/types/v2/tweet.v2.types.ts

```diff
@@ -22,6 +22,8 @@ export interface TweetV2TimelineParams extends Partial<Tweetv2FieldsParams> {
 export interface Tweetv2SearchParams extends TweetV2TimelineParams {
   previous_token?: string;
   query: string;
+  sort_order?: 'recency' | 'relevancy';
+  max_results?: number;
```

Comment:
max_results is already in TweetV2TimelineParams, so let's remove it from there

But awesome for the sort_order, thank you 🙏